public/app/features/alerting/unified/components/AlertLabelDropdown.tsx

```diff
@@ -10,6 +11,36 @@ export interface AlertLabelDropdownProps {
   defaultValue?: SelectableValue;
   type: 'key' | 'value';
 }
+const _customFilter = createFilter({ ignoreCase: false });
+function customFilter(opt: SelectableValue, searchQuery: string) {
+  return _customFilter(
+    {
+      label: opt.label ?? '',
+      value: opt.value ?? '',
+      data: {},
+    },
+    searchQuery
+  );
+}
+
+const handleIsValidNewOption = (
+  inputValue: string,
+  value: SelectableValue<string> | null,
+  options: OptionsOrGroups<unknown, GroupBase<unknown>>
```

Comment:
nit: Isn't it a list of `SelectableValue<string>`? maybe we could provide a better type than `unknown` and could remove `isWithLabel` checking

Reply:
I tried but then the Select component is complaining about that